File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/ticket_item/exception/InvalidTicketItemException.java
package band.gosrock.domain.domains.ticket_item.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class InvalidTicketItemException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new InvalidTicketItemException();

    private InvalidTicketItemException() {
        super(TicketItemErrorCode.INVALID_TICKET_ITEM);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/ticket_item/exception/InvalidTicketItemException.kt
package band.gosrock.domain.domains.ticket_item.exception

import band.gosrock.common.exception.DuDoongCodeException

class InvalidTicketItemException private constructor() : DuDoongCodeException(TicketItemErrorCode.INVALID_TICKET_ITEM) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = InvalidTicketItemException()
    }
}
